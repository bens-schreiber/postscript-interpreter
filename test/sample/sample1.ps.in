1 1 add
1 2 sub
2 2 mul
2 4 mod
4 2 div
(four) length
(a) 0 get
(abcd) 0 2 getinterval
(xxlo) 0 (he) putinterval
1 2 eq
2 2 eq
(hi) (hi) eq
true false ne
true true and
1 2 and
100 200 or
5 not
1 2 gt
1 2 lt
5 dict
5 dict length
5 dict maxlength
5 dict begin
end
/myvar (foobar) def
myvar




