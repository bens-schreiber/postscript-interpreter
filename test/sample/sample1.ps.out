(foobar)
5
0
--nostringval--
true
false
-6
236
0
true
true
true
true
false
(hello)
(ab)
97
4
2
2
4
-1
2